### PR TITLE
feat: adds icons to indicate searchable, hoverable and/or editable layer

### PR DESCRIPTION
**Pull request by AmandaTamanda**
Before:
Bildschirmfoto vom 2024-12-04 11-18-42

After:
image

image

### CODE REVIEW

**Review by FritzHoing — APPROVED** [review]

File: src/components/ToolMenu/LayerTree/index.tsx [hunk 1]
```diff
@@ -221,6 +232,24 @@ export const LayerTree: React.FC<LayerTreeProps> = ({
...
+                  icon={faCircleInfo}
+                  className='layer-icon'
+                />
+              )}
+              {layer.get('editable') && shouldDisplayIcon && (
+                < FontAwesomeIcon
+                  icon={faPen}
+                  className='layer-icon'
+                />
+              )}
```
Comment (dnlkoch) [inline src/components/ToolMenu/LayerTree/index.tsx:252]: The current placement will move the loading indicator (see `loading-dots`) to the right and after the newly added icons. Is this intended?

File: src/components/ToolMenu/LayerTree/index.tsx [hunk 2]
```diff
@@ -221,6 +232,24 @@ export const LayerTree: React.FC<LayerTreeProps> = ({
...
+              {layer.get('hoverable') && shouldDisplayIcon && (
+                < FontAwesomeIcon
+                  icon={faCircleInfo}
+                  className='layer-icon'
+                />
+              )}
+              {layer.get('editable') && shouldDisplayIcon && (
+                < FontAwesomeIcon
```
Comment (dnlkoch) [inline src/components/ToolMenu/LayerTree/index.tsx:248]:
```suggestion
                <FontAwesomeIcon
```

File: src/components/ToolMenu/LayerTree/index.tsx [hunk 3]
```diff
@@ -221,6 +232,24 @@ export const LayerTree: React.FC<LayerTreeProps> = ({
...
             >
               {layer.get('name')}
+              {layer.get('searchable') && shouldDisplayIcon && (
+                < FontAwesomeIcon
+                  icon={faMagnifyingGlass}
+                  className='layer-icon'
+                />
+              )}
+              {layer.get('hoverable') && shouldDisplayIcon && (
+                < FontAwesomeIcon
```
Comment (dnlkoch) [inline src/components/ToolMenu/LayerTree/index.tsx:242]:
```suggestion
                <FontAwesomeIcon
```

File: src/components/ToolMenu/LayerTree/index.tsx [hunk 4]
```diff
@@ -221,6 +232,24 @@ export const LayerTree: React.FC<LayerTreeProps> = ({
               className='layer-name'
             >
               {layer.get('name')}
+              {layer.get('searchable') && shouldDisplayIcon && (
+                < FontAwesomeIcon
```
Comment (dnlkoch) [inline src/components/ToolMenu/LayerTree/index.tsx:236]:
```suggestion
                <FontAwesomeIcon
```

File: src/components/ToolMenu/LayerTree/index.tsx (hunk 1 above)
Comment (AmandaTamanda) [reply src/components/ToolMenu/LayerTree/index.tsx:252]: This was not intended, i'll remedy that

**Comment by AmandaTamanda** [comment]
@FritzHoing @dnlkoch thank you for reviewing!

**Pull request merged**